MainModule/Client/Core/Remote.lua

```diff
@@ -505,6 +509,29 @@ return function(Vargs, GetEnv)
 			local ms = ((math.floor((t2-t)*mult+0.5)/mult)*1000)
 			return ms
 		end;
+				
+		FPS = function()
+			local FPS = Remote.Get("FPS")
+			if not FPS then return false end			
+			local startTime = os.clock() 
+			local X = 1 
+			local FPS_Counter = 0
+			
+			local RenderStepped = service.RunService.RenderStepped
+			local Heartbeat = service.RunService.Heartbeat
+
+			local function GetFPS()
+				FPS_Counter+=1
+				if (os.clock() - startTime) >= X  then 
+					local fps = math.floor(FPS_Counter / (os.clock() - startTime))
+					return fps
+					FPS_Counter = 0
+					startTime = os.clock()
+				end
+			end
+			local FPS = Heartbeat:Connect(GetFPS)
+			return FPS
+		end;
```

Comment:
What this basically does is that this only checks the checks the HeartBeat FPS which is the same as the physics FPS.
In which case you should replace this function with `workspace:GetRealPhysicsFPS()`.